.eslintrc.json

```diff
@@ -8,15 +8,13 @@
     "plugin:jsx-a11y/recommended",
     "plugin:@next/next/recommended",
     "plugin:react/recommended",
-    "prettier",
-    "prettier/flowtype",
-    "prettier/react"
+    "prettier"
   ],
   "env": {
     "browser": true,
     "es6": true
   },
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
`babel-eslint` is deprecated, replaced by this parser. Upgrading is necessary to work with eslint 8.x.x